ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_searching_sorting.py

```diff
@@ -337,3 +337,33 @@ def test_jax_numpy_sort_complex(
         a=x[0],
         test_values=False,
     )
+
+
+# where
+@handle_frontend_test(
+    fn_tree="jax.numpy.where",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("numeric")),
+    test_with_out=st.just(False),
+)
+def test_jax_numpy_where(
+    test_flags,
```

Comment:
you weren't supposed to remove `dtype_and_x`,
you can add this argument back in and also make the arguments keyword only by adding `*` first.